### DISCUSSION TITLE
Renovate opened 2 similar PR's in two different repos, but with different PR title formats.

### DISCUSSION THREAD

**Discussion by shadycuz** [renovatebot/renovate#22898] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

This isn't a problem, but something I noticed. I have two similar repos and Renovate just made a PR in each of them, updating the same package but with different PR titles? 

![image](https://github.com/renovatebot/renovate/assets/17090999/39a3f365-3f2e-49f9-ba91-ba9da754f3b4)

I'm not sure why it would have done that. I use poetry for both tools and I think they have similar renovate configs. Just thought I would let you know.

### What you tried so far.

Took a quick glance at the package manager for each repo (both poetry), and the renovate config file and they look similar.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by viceice** [comment, marked as answer]
it's probably because you use semantic commit inside one of them

**Reply by shadycuz** [reply]
Haha coolio, I accidentally used a semantic commit , will checkout the docs, thanks

**Comment by rarkins** [comment]
see config option semanticCommits if you want to control it manually

**Comment by HonkingGoose** [comment]
Relevant quote from our docs: [^1]

> Renovate looks at the last 10 commit messages in the base branch to decide if the repository uses semantic commits. If there are semantic commits, Renovate uses the [conventional-commits-detector](https://github.com/conventional-changelog/conventional-commits-detector) to decide what convention the commit messages follow.

[^1]: [Renovate docs, semantic commits](https://docs.renovatebot.com/semantic-commits/)
